sklearn/preprocessing/tests/test_imputation.py

```diff
@@ -357,3 +374,408 @@ def test_imputation_copy():
 
     # Note: If X is sparse and if missing_values=0, then a (dense) copy of X is
     # made, even if copy=False.
+
+
+"""--------------------- BEGIN KNNIMPUTER TEST ---------------------"""
```

Comment:
Using a string like this is fairly unconventional. Cf section headings in sklearn/tests/test_base.py